website/content/en/highlights/2021-11-16-implicit-namespacing.md

```diff
@@ -0,0 +1,70 @@
+---
+date: "2021-11-16"
+title: "Automatic namespacing for config files"
+description: "A guide that addresses the new automatic namespacing functionality"
+authors: ["barieom"]
+pr_numbers: []
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+---
+
+# Automatic namespacing for configuration
+
+
+We've released automatic namespacing for configuration files, which simplifies namespace configuration for your pipelines.
+
+As Vector continues to evolves, releasing more configuration-heavy functionality such as an aggregator role and pipelines features, there's often a proliferation of configurations necessary to program Vector. Even more, the ability to organize Vector across multiple files was previously lacked clarity and includes a heavy amount of boilerplate, which further made configuration difficult for collaboration and navigation.
+
+To further paint this pain point, Vector users may have had dozens of Vector configuration files, from multiple source files to countless sink files, in a single directory (let's assume Vector is loaded using the configuration `--config-dir /etc/vector`):
+```
+/etc/vector/
+│   file001.toml
+│   file002.toml    
+│	file003.toml
+│   ...
+│   file022.toml
+│   file023.toml
+```
+
+To solve this issue, _automatic namespacing_ provides Vector users the ability to organize their configuration into separate files based on Vector's configuration directory structure. This will make it easy for users like you to split up your configuration files and collaborate with others on their team. Vector will look in every subfolder for any component configuration files and use filenames as their component ID. 
```

Comment:
In general, "enables" reads better than "provides ... the ability"

Reply:
Also, in general I think the documentation currently over uses "will" constructions. We should opt to use the present tense as much as possible. In this case, "This will make it easy..." should be "This makes it easy..."